### PR TITLE
Greenplum 7 README for Rocky8 to build OSS GPDB

### PR DESCRIPTION
**Pull request by mounikagunnam**
Authored-by : Mounika M <mmounika@vmware.com>

Description:

This script enables users to initiate the gpdb cluster on Rocky 8 Platform without any hassles.

1. It installs all the packages and its dependencies required for gpdb7.
2. It also sets the OS and Kernel level settings that are optimum for gpdb7.

## Here are some reminders before you submit the pull request
- [ ] Add tests for the change
- [ ] Document changes
- [ ] Communicate in the mailing list if needed
- [ ] Pass `make installcheck`
- [ ] Review a PR in return to support the community

### CODE REVIEW

File: README.Rocky8.bash [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    echo "vm.dirty_background_bytes = 1610612736 # 1.5GB" | sudo tee -a /etc/sysctl.d/10-gpdb.conf > /dev/null
+    echo "vm.dirty_bytes = 4294967296 # 4GB" | sudo tee -a /etc/sysctl.d/10-gpdb.conf > /dev/null
+fi
+
+sudo tee -a /etc/security/limits.d/10-nproc.conf << EOF
+* soft nofile 524288
+* hard nofile 524288
+* soft nproc 131072
+* hard nproc 131072
+soft  core unlimited
```
Comment (Shakarada) [inline README.Rocky8.bash:94]: * soft core unlimited 
Add * before the line

File: README.Rocky8.bash [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+EOF
+
+sudo tee -a /etc/fstab << EOF
+/dev/data /data xfs nodev,noatime,inode64 0 0
+EOF
+
+sudo sysctl -p
```
Comment (Shakarada) [inline README.Rocky8.bash:101]: add after the sysctl configuration

File: README.Rocky8.bash [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+* soft nofile 524288
+* hard nofile 524288
+* soft nproc 131072
+* hard nproc 131072
+soft  core unlimited
+EOF
+
+sudo tee -a /etc/fstab << EOF
+/dev/data /data xfs nodev,noatime,inode64 0 0
+EOF
```
Comment (deenav) [inline README.Rocky8.bash:99]: I don't think we need to update the `/etc/fstab` file with filesystem details. Is it mandatory if so How to determine the data disk getting used is `/dev/data`

**Review by Shakarada — APPROVED** [review]
LGTM

**Review by deenav — APPROVED** [review]
+1

**Review by 50wu — APPROVED** [review]
+1

**Pull request merged**